site/blog/2024-06-26-datapackage-v2-release/README.md

```diff
@@ -0,0 +1,45 @@
+
+If you are curious about the conversations we had during the Standard review,they were all captured (and recorded) in [the blog summaries of the community calls](https://frictionlessdata.io/blog/). Alternatively you can also check out the [closed issues on GitHub](https://github.com/frictionlessdata/datapackage/milestone/6?closed=1). 
+
+## So what is new in version 2?
+
+During these months we have been working on the core specifications that compose the Standard, namely: [Data Package](https://datapackage.org/standard/data-package/) – a simple container format for describing a coherent collection of data in a single ‘package’, [Data Resource](https://datapackage.org/standard/data-resource/) to describe and package a single data resource, [Table Dialect](https://datapackage.org/standard/table-dialect/) to describe how tabular data is stored in a file, and [Table Schema](https://datapackage.org/standard/table-schema/) to declare a schema for tabular data.
+
+During the update process we tried to be as little disruptive as possible, avoiding breaking changes when possible. 
+
+We put a lot of effort into removing ambiguity, cutting or clarifying under-defined features, and promoting some well-oiled [recipes](https://datapackage.org/recipes/caching-of-resources/) into the Standard itself. Among other things, we added versioning, to ensure consistency and reliability over time, and added support for categorical data.
+
+If you are curious and would like to know more details about what changes with version 2, go and have a look at the [Changelog](https://datapackage.org/overview/changelog/) we published.
+
+To increase and facilitate adoption, we published a [metadata mapper written in Python](https://github.com/frictionlessdata/dplib-py). We have also worked on Data Package integrations for the most notable open data portals out there. We therefore proposed a Data Package serializer to [Invenio RDM](https://inveniordm.web.cern.ch/), created a pull request that exposes `datapackage.json` as a metadata export target in the [Open Science Framework](https://www.cos.io/) system, and built an extension that adds a `datapackage.json` endpoint to every dataset in [CKAN](https://github.com/frictionlessdata/ckanext-datapackage).
```

Comment:
I would mention Invenio RDM / Zenodo together

Reply:
Thanks for the super useful feedback @roll. I updated the blog to integrate them, let me know what you think.